policies/committer-policy.md

```diff
@@ -0,0 +1,112 @@
+
+# Policy for OpenSSL Committers
+
+## Who is a committer?
+
+OpenSSL committers are contributors who have commit access to the OpenSSL
+source code repository.  Committers review and commit their own patches
+as well as those of other contributors.
+
+## How to become a committer?
+
+Commit access is granted by a vote of the [OMC] typically on the
+recommendation of the [OTC] (see the [OpenSSL Bylaws]).
+
+We welcome contributors who become domain experts in some part of
+the library (for example, low-level crypto) as well as generalists
+who contribute to all areas of the codebase.  All committers share
+the responsibility for the overall health of the project: aside from
+contributing quality features, committers are team players who fix bugs,
+address open issues, review community contributions, and improve tests
+and documentation.  Committers are also shepherds of the OpenSSL community
+and its code of conduct.
+
+To become a committer, start by contributing code.  Read our coding style,
+and get to know our build and test system.  Then, use the Github issue
+tracker, and our mailing lists to find impactful ideas to work on.
+
+## How to maintain committer status?
+
+To maintain committer status, you must stay active in the project.  Refer
+to the OpenSSL Bylaws for details.
+
+In the unlikely and unfortunate event that your actions conflict with
+the project objectives or are otherwise disruptive, committer status
+may also be revoked by the OMC.
+
+## Approvals and code reviews
+
+All submissions must be reviewed and approved by at least two committers,
+one of whom must also be an OTC member.  Neither of the reviewers can
+be the author of the submission.
+
+The sole exception to this is during the release process where the
+author's review does count towards the two needed for the automated
+release process and [NEWS] and [CHANGES] file updates.
+
+In the case where two committers make a joint submission, they can review
+each other's code but not their own.  A third reviewer will be required.
+
+An OMC member may apply a _hold: needs OMC decision_ label to a submission.
+An OTC member may apply a _hold: needs OTC decision_ to a submission.
```

Comment:
Didn't think of this until now (so probably something to think about after the vote) - but do we mean to restrict things so that only OMC can place an OMC hold and only OTC can place an OTC hold? Shouldn't it be possible for any committer to refer a decision to OMC or OTC if they think it is appropriate?

Reply:
Only OMC/OTC members should be able to _force_ an OMC/OTC vote by placing a hold. Anyone else (committers or normal community members) still has the possibility to convince one or more OMC/OTC members of his opinion and get them to place the hold.